server/controllers/configurationsController/index.js

```diff
@@ -57,50 +55,48 @@ const ConfigurationsController = {
     }
   },
   index: async (req, res) => {
-    try {
-      const dataSet = new Set()
-      const { appDb } = req.app.locals
-      const { uid } = req.params
-      const streamConfigurations = await ConfigModel.index(appDb, uid)
+    const cachedUsers = new Map()
+    const dataSet = new Set()
+    const { appDb } = req.app.locals
+    const { uid } = req.params
+    const activeOnly = Object.hasOwn(req.query, 'activeOnly')
```

Comment:
it'll be more extensible to have this be an attribute like 'status' that we can use to toggle by other values too, like `?status=active` or `?status=inactive`. For now all we'll need to account for is the active status but in the future it will be easier to add others.

Reply:
[2afe7af](https://github.com/AMP-SCZ/dpdash/pull/739/commits/2afe7af8b409f3ec568dc3f205c9af37c68d334e)